### DISCUSSION TITLE
[Q&A] Communication error with fl_admin

### DISCUSSION THREAD

**Discussion by jingyunc** [NVIDIA/NVFlare#2184] [Q&A] [answered]
### Python version (`python3 -V`)

3.8

### NVFlare version (`python3 -m pip list | grep "nvflare"`)

2.4.0rc3

### Operating system

Ubuntu 20.04

### Have you successfully run any of the following examples?

- [ ] hello-numpy-sag with simulator
- [ ] hello-pt with simulator
- [ ] hello-numpy-sag with POC
- [ ] hello-pt with POC

### Please describe your question

I created the default startup kit with `nvflare provision` and successfully started the server.

However, when launching fl_admin.sh and typing in the admin email, I got following error:

```
User Name: my@email.com
Trying to obtain server address
Obtained server address: server1:8003
Trying to login, please wait ...
Trying to login, please wait ...
Trying to login, please wait ...
Trying to login, please wait ...
Trying to login, please wait ...
Communication Error - please try later
```

The email was specified in project.yml and the admin folder was named after it.

I tried launching fl_admin.sh from different or same terminal session as the sever, and the error was same.

**Comment by YuanTingHsieh** [comment, marked as answer]
Same as in https://github.com/NVIDIA/NVFlare/discussions/2183
Your DNS should be able to find server1.

You could either change this to "localhost" in project.yml and it will work.
Or in the real-world use case, your server needs to have a valid name, say like "hello@aws.com"
And you need to make sure all of your client sites and the machine you run the admin console can find this "hello@aws.com"

**Reply by YuanTingHsieh** [reply]
another way just as a hack is like this, you can edit your /etc/host file:

`sudo vim /etc/hosts`

and add an entry

```
127.0.0.1 view-localhost
127.0.0.1 localhost
127.0.0.1 server1
```
